voice/receive-an-inbound-call.go

```diff
@@ -0,0 +1,27 @@
+package main
```

Comment:
Since we have this as an option for ADP snippets, we could have it work there as well if we follow https://github.com/Nexmo/code-snippet-specs/blob/master/voice/receive-an-inbound-call.md